cmd/agent/dist/conf.d/jmx.d/conf.yaml.example

```diff
@@ -20,11 +20,14 @@ instances:
   #   tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar # To be set when process_name_regex is set
 
   #   name: jmx_instance
-  #   # java_bin_path: /path/to/java # Optional, should be set if the agent cannot find your java executable
-  #   # java_options: "-Xmx200m -Xms50m" # Optional, Java JVM options
-  #   # trust_store_path: /path/to/trustStore.jks # Optional, should be set if ssl is enabled
-  #   # trust_store_password: password
-  #   # refresh_beans: 600 # Optional (in seconds), default is 600 seconds. Sets refresh period for refreshing matching MBeans list.
+  #   java_bin_path: /path/to/java # Optional, should be set if the agent cannot find your java executable
+  #   java_options: "-Xmx200m -Xms50m" # Optional, Java JVM options
+  #   trust_store_path: /path/to/trustStore.jks # Optional, should be set if "com.sun.management.jmxremote.ssl" is set to true.
```

Comment:
`is set to true` on the target JVM I assume?

may require a longer explanation on what this `jks` file is exactly (not necessarily here though, full explanation could be on docs website)